src/phoebus_utils/root_find.hpp

```diff
@@ -29,9 +29,10 @@ struct RootFind {
     if (ya * yb > 0.0) {
       if (status != nullptr) {
         *status = RootFindStatus::failure;
+      } else {
+        printf("Root not bracketed in find_root. a, b, ya, yb = %g %g %g %g\n", a, b, ya,
```

Comment:
Since in the radiation force solver I start with a relatively small root bracket and then if that fails I try again with a larger bracket for a while, we don't want this `printf` statement to fire unless we're not (potentially) catching rootfind errors elsewhere with the `status` argument.